json-endpoints/jetpack/class.jetpack-json-api-endpoint.php

```diff
@@ -68,7 +68,19 @@ protected function validate_call( $_blog_id, $capability, $check_manage_active =
 			return $error;
 		}
 
-		if ( $check_manage_active &&  'GET' !== $this->method && ! Jetpack::is_module_active( 'manage' ) ) {
+		if (
+			$check_validation &&
+			'GET' !== $this->method &&
+			/**
+			 * Filter to disallow JSON API requests to the site.
+			 * Setting to false will not allow you to manage your site remotely from WordPress.com and disallow plugin auto-updates.
+			 *
+			 * @since 6.0.0
```

Comment:
Is the 6.0.0 version intended or does it need a version bump?

Reply:
probably :)

Reply:
```suggestion
			 * @since 7.2.0
```